packages/core/src/app/payment/paymentMethod/MolliePaymentMethod.tsx

```diff
@@ -100,4 +110,4 @@ const MolliePaymentMethod: FunctionComponent<MolliePaymentMethodsProps & WithInj
         />);
 };
 
-export default withHostedCreditCardFieldset(MolliePaymentMethod);
+export default withHostedCreditCardFieldset(withLanguage(MolliePaymentMethod));
```

Comment:
Instead of doing this you should be able to use `const localeContext = useContext(LocaleContext);` in the component itself. For example you can have a look at this [component](https://github.com/bigcommerce/checkout-js/blob/master/packages/core/src/app/payment/paymentMethod/ApplePayPaymentMethod.tsx#L14)